Detective outfits

Davyei opened the pull request:
<!--
You are making this pull request for the Moffstation fork of Space Station 14.

Please be sure to follow general guidelines for upstream PRs, but also be sure to follow the Moffstation guidelines.
Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline
See the Harmony contributing guidelines for an example on what we want: https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md
-->

## About the PR
<!-- What did you change? -->
Adds Six new outfits items to the detective & resprites the Detective's Vest
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Detective deserved more Clothes for cool gimmicks & stuff
Detective's vest needed a resprite due to being incredibly outdated.
## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="529" height="384" alt="Screenshot 2025-10-06 232305" src="https://github.com/user-attachments/assets/b2373f2c-ffee-42f2-bb37-1fdbd31005a5" />
<img width="704" height="622" alt="Screenshot 2025-10-07 010511" src="https://github.com/user-attachments/assets/6d4af4cc-7b4b-493f-b009-62bae237b00f" />

<img width="997" height="816" alt="Screenshot 2025-10-07 003828" src="https://github.com/user-attachments/assets/53fbccb2-48b9-439b-9f7e-7fe82e829b17" />
<img width="643" height="944" alt="Screenshot 2025-10-07 010002" src="https://github.com/user-attachments/assets/b4f864a2-15ca-420b-9508-ff31e9210f79" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X ] I have properly sectioned my changes into fork namespaces, and/or followed proper guidelines for modifying upstream files.
- [X ] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: New Detective Cosmetics

Davyei committed:
Outfits are in the game!!

The Sleuth Outfit
The Investigator Outfit
Detective's Badge
Detective's Vest Resprite

All Added & exist in the game!

Davyei committed:
Added to Loadouts

Add all the Detective Outfits to the Loadout Options

Davyei committed:
DetDrobe update!

Adds the items to the Detdrobe

Davyei committed:
I forgot to do anything with this so bye bye.

I actually don't need this & forgot to remove it. :P

Davyei committed:
Sleuth skirt Test fail fix

This should fix the Sleuth skirt test fail.

Davyei committed:
Detvest test fail fix

This hopefully will fix the Detective Vest Failing

Davyei committed:
SLEUTH SUIT FIX!!

I LOVE CAPITALISATION!!

I missed one of the lines needing to be capitalised in the last sleuth suit  fix. . . the woes of trying to fix this at 1:40am.

Davyei committed:
INVESTIGATOR SKIRT TEST FAIL FIX

I LOVE CAPITALISATION SO SO VERY MUCH

NOW EVERYTHING SHOULD WORK & IF IT DOESN'T I AM GOING TO SLEEP CAUSE ITS ALMOST 2AM. >:(

Davyei committed:
FINAL CAPITALISATION FIX

I HATE CAPITAL LETTERS

Hopefully all tests pass now & github doesn't pull a columbo & say 'Just one more thing. . .'

Wintoli commented:
These are all great, but honestly I'd probably keep the old detective vest? It was super recognizable and iconic with the little yellow mark that matched the flask. The replacement one is a bit more muted and easily mistakable as any other old vest or maybe even military gear. But mainly just cause I thought it was a good sprite already.

southbridge-fur commented:
Why is Investigator capitalized?

Davyei committed:
Detective Cabinet New Fill +Removing ALL capitalisation!

Detective's Cabinet Fill now randomly pics an outfit & coat, it also always contains the det's badge

HOPEFULLY NOW!! The Tests won't fail.

Davyei committed:
I LOVE CAPITAL LETTERS, PLEASE TELL ME THIS WORKS

AAAAAAAAAAAAAAAAAAHHHHH

Davyei committed:
THIS SHOULD FIX IT, PLEASE WORK

WHY DOES "\" & "/" MATTER HERE, DO THEY MATTER?!? DOES ANYTHING MATTER?!!?

Davyei committed:
License updates for Centronias

I noticed I forgot to update the meta.json files to be 4.0

also somehow missed two more capital letters

Davyei committed:
Callmore my saviour

I was using \ instead of / without realising that that completely bricks Linux :P

Davyei committed:
Callmore my linux saviour

we love \

another reason Windows is superior B)

Callmore commented:
RSI folders start with a capital letter, should be lowercase.

Davyei committed:
.rsi capitalisation fix part one!

Davyei committed:
Capitalisation finally fixed!!

GOD PLEASE WORK

Davyei committed:
.rsi jumpsuit fix part one

Davyei committed:
Jumpsuit fix :D

PLEASE BE OVER, EVERYTHING SHOULD WORK NOW. NO MORE CAPITAL LETTERS

Davyei committed:
Merge branch 'master' into Detective-Outfits

Signed-off-by: Dave <davidaperry2@gmail.com>

DuckManZach commented on Resources/Textures/_Moffstation/Clothing/Neck/detective-badge.rsi/meta.json:
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```
``` suggestion
  "license": "CC-BY-SA-4.0",
```

Davyei commented on Resources/Prototypes/_Moffstation/Entities/Clothing/OuterClothing/coats.yml:
```diff
@@ -33,6 +33,24 @@
+    contents:
+    - id: CigarGold
+    - id: CigarGold
+    - id: FlippoEngravedLighter
```
the sleuth coat specifically has this because it is a reference to columbo who smokes cigars hence why the sleuth coat has a different fill to the other coats

Davyei commented on Resources/Prototypes/_Moffstation/Entities/Clothing/OuterClothing/coats.yml:
```diff
@@ -33,6 +33,24 @@
+    - id: CigarGold
+    - id: FlippoEngravedLighter
+  - type: ExplosionResistance
+    damageCoefficient: 1 #its a coat. it doesnt do shit # Moffstation - End (New Detective Outfits)
```
same with the comment above too

Davyei committed:
Update Resources/Textures/_Moffstation/Clothing/Neck/detective-badge.rsi/meta.json

Co-authored-by: DuckManZach <144298822+DuckManZach@users.noreply.github.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei commented on Resources/Prototypes/_Moffstation/Entities/Clothing/Uniforms/jumpsuits.yml:
```diff
@@ -171,6 +171,28 @@
+  - type: Sprite
+    sprite: _Moffstation/Clothing/Uniforms/Jumpsuit/investigator.rsi
+  - type: Clothing
+    sprite: _Moffstation/Clothing/Uniforms/Jumpsuit/investigator.rsi
```
this is sorted, github sorted it for me when another pr got merged but git didn't add the space.

Davyei committed:
Lots of changes Yippeee

These are the changes Fried said to make

Davyei committed:
Lots of changes Yippeee

These are the changes Fried said to make
